staging/src/k8s.io/apimachinery/pkg/util/wait/poll.go

```diff
@@ -41,7 +41,7 @@ func PollUntilContextCancel(ctx context.Context, interval time.Duration, immedia
 //	err := PollUntilContextCancel(ctx, interval, immediate, condition)
 //
 // The deadline context will be cancelled if the Poll succeeds before the timeout, simplifying
-// inline usage. All other behavior is identical to PollWithContextTimeout.
+// inline usage. All other behavior is identical to PollUntilContextTimeout.
```

Comment:
Is the last part of this comment necessary? I find it confusing because this func *is* `PollUntilContextTimeout`